src/tools/tidy/src/ui_tests.rs

```diff
@@ -46,8 +46,9 @@ fn check_entries(tests_path: &Path, bad: &mut bool) {
             );
         }
     }
-    if ENTRY_LIMIT > max {
-        tidy_error!(bad, "`ENTRY_LIMIT` is too high (is {ENTRY_LIMIT}, should be {max})");
+    let max_add = max + (max * 15) / 100;
```

Comment:
The point is: I don't want to reduce the limit in this PR because I don't really want to force others to organize unrelated tests just because they want to add one test to the directory that unfortunately is the one with the highest number of entries. That's the problem here: adding a test to any other directory does not exhibit the behaviour that I run into here, but `parser/` is the directory the test this PR adds has to live in (unless you have better suggestions). There is no better place to put it, as `parser/` itself only has few subdirectories of its own.

Do you want me to organize unrelated tests (like done now) and just remove the limit, or do you want me to increase the limit?

Reply:
I know why `ui/issues` and `ui` root have limits - because we want to eliminate "unclassified" tests entirely (so  the limits will eventually decrease to 0).

The only reason to limit other, "meaningful" directories like `parser` is GitHub interface, AFAIK.
https://github.com/rust-lang/rust/issues/73494 says GitHub allows up to 1000 entries, so `ENTRY_LIMIT` can be set to 1000, I think.